gallery/plot_datapoints.py

```diff
@@ -209,42 +205,78 @@ def get_transform(train):
 # I had a Datapoint but now I have a Tensor. Help!
 # ------------------------------------------------
 #
-# For a lot of operations involving datapoints, we cannot safely infer whether
-# the result should retain the datapoint type, so we choose to return a plain
-# tensor instead of a datapoint (this might change, see note below):
+# By default, operations on :class:`~torchvision.datapoints.Datapoint` objects
+# will return a pure Tensor:
 
 
 assert isinstance(bboxes, datapoints.BoundingBoxes)
 
 # Shift bboxes by 3 pixels in both H and W
 new_bboxes = bboxes + 3
 
-assert isinstance(new_bboxes, torch.Tensor) and not isinstance(new_bboxes, datapoints.BoundingBoxes)
+assert isinstance(new_bboxes, torch.Tensor)
+assert not isinstance(new_bboxes, datapoints.BoundingBoxes)
+
+# %%
+# .. note::
+#
+#    This behavior only affects native ``torch`` operations. If you are using
+#    the builtin ``torchvision`` transforms or functionals, you will always get
+#    as output the same type that you passed as input (pure ``Tensor`` or
+#    ``Datapoint``).
 
 # %%
-# If you're writing your own custom transforms or code involving datapoints, you
-# can re-wrap the output into a datapoint by just calling their constructor, or
-# by using the ``.wrap_like()`` class method:
+# But I want a Datapoint back!
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# You can re-wrap a pure tensor into a datapoint by just calling the datapoint
+# constructor, or by using the ``.wrap_like()`` class method (see more details
+# above in :ref:`datapoint_creation`):
 
 new_bboxes = bboxes + 3
 new_bboxes = datapoints.BoundingBoxes.wrap_like(bboxes, new_bboxes)
```

Comment:
non-blocking thought to reduce the number of chars to type, it could be simple to have `bboxes.wrap_like(new_bboxes)` or `bboxes.wrap_to(new_bboxes)`.
I think we already thought about that earlier but failed to get a good naming ...

Reply:
Yeah, I find it a bit long and clunky as well.

```
bboxes.wrap_to(new_bboxes).
```

That would be the best UX but unfortunately we can't do that because `wrap_to` would have to be a method on pure Tensors. Maybe we could override `to()`... ?

Reply:
We have #7780 to go from `BoundingBoxes` to `BBoxes`. I'm somewhat against doing `bboxes.wrap_like(...)` here, since it no longer makes it clear that this is a class and not instance method.